packages/features/bookings/lib/handleConfirmation.ts

```diff
@@ -324,6 +327,7 @@ export async function handleConfirmation(args: {
       const eventTypeSlug = updatedBookings[index].eventType?.slug || "";
       const evtOfBooking = {
         ...evt,
+        rescheduleReason: updatedBookings[index].cancellationReason || null,
```

Comment:
Shouldn't it be .rescheduleReason?

Reply:
The reschedule reason is saved in the cancellationReason field of the newly created rescheduled booking record.